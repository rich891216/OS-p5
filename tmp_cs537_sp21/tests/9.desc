[mencrypt, (dump_rawphymem, getpgtable)] Call mencrypt on partially encrypted page range.
