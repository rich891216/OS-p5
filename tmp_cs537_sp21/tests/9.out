XV6_TEST_OUTPUT Index 0: pdx: 0x0, ptx: 0xB, present: 0, writable: 1, encrypted: 1
XV6_TEST_OUTPUT Index 1: pdx: 0x0, ptx: 0xA, present: 0, writable: 1, encrypted: 1
XV6_TEST_OUTPUT Index 2: pdx: 0x0, ptx: 0x9, present: 0, writable: 1, encrypted: 1
XV6_TEST_OUTPUT Index 3: pdx: 0x0, ptx: 0x8, present: 0, writable: 1, encrypted: 1
XV6_TEST_OUTPUT Index 4: pdx: 0x0, ptx: 0x7, present: 1, writable: 1, encrypted: 0
XV6_TEST_OUTPUT Index 0: pdx: 0x0, ptx: 0xB, present: 0, writable: 1, encrypted: 1
XV6_TEST_OUTPUT Index 1: pdx: 0x0, ptx: 0xA, present: 0, writable: 1, encrypted: 1
XV6_TEST_OUTPUT Index 2: pdx: 0x0, ptx: 0x9, present: 0, writable: 1, encrypted: 1
XV6_TEST_OUTPUT Index 3: pdx: 0x0, ptx: 0x8, present: 0, writable: 1, encrypted: 1
XV6_TEST_OUTPUT Index 4: pdx: 0x0, ptx: 0x7, present: 0, writable: 1, encrypted: 1
